src/core/components/Conversation.tsx

```diff
@@ -54,13 +62,18 @@ const ConversationComponent = ({ messages, onRendered }: ConversationProps) => {
 
         if (message.role === 'assistant' && message.content) {
           return (
-            <AssistantMessage key={index} content={message.content as string} />
+            <AssistantMessage
+              key={index}
+              content={message.content as string}
+              // @ts-expect-error reasoningContent may exist
+              reasoningContent={message.reasoningContent}
```

Comment:
I think you could avoid expecting this error if we did some check on the message before passing its possible reasoning content

```
reasoningContent={
  message.role === 'assistant' && 'reasoningContent' in message
    ? message.reasoningContent
    : undefined
}
  ```